wrangler-core/src/main/java/io/cdap/directives/column/SetType.java

```diff
@@ -108,4 +113,17 @@ public Mutation lineage() {
       .relation(col, col)
       .build();
   }
+
+  @Override
+  public Relation transform(RelationalTranformContext relationalTranformContext,
+                            Relation relation) {
+    java.util.Optional<ExpressionFactory<String>> expressionFactory = SqlExpressionGenerator
+            .getExpressionFactory(relationalTranformContext);
+    if (!expressionFactory.isPresent()) {
+      return new InvalidRelation("Cannot find an Expression Factory");
+    }
+    String expression = SqlExpressionGenerator.getColumnTypeExp(type, col, scale);
```

Comment:
Some of the directive specific methods are in their respective classes and some are in the SqlExpressionGenerator. We should try to maintain consistency